plugins/azure/virtualmachines/vmssTrustedLaunchEnabled.js

```diff
@@ -0,0 +1,53 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Scale Sets Trusted Launch Enabled',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that Trusted launch security option is enabled for virtual machine scale sets.',
+    more_info: 'Trusted launch protects against advanced and persistent attack techniques. Trusted launch is composed of several, coordinated infrastructure technologies that can be enabled independently. Each technology provides another layer of defense against sophisticated threats.',
+    recommended_action: 'Modify VMSS configurations and enable trusted launch.',
```

Comment:
can we enable trsuted launch on existing vm scale set?
<img width="862" alt="Screenshot 2023-12-28 at 8 19 31 PM" src="https://github.com/aquasecurity/cloudsploit/assets/113519745/69368a30-5a73-4611-847e-d17383815352">